assets/sass/patterns/organisms/hero-banner.scss

```diff
@@ -0,0 +1,206 @@
+@import "../../definitions";
+
+.hero-banner__picture {
+  display: none;
+}
+
+.js {
+  .hero-banner-item {
```

Comment:
There is no reason to have any styles on the hero banner that are reliant on javascript. `.js` is only introduced on the page when javascript is enabled.

<img width="1680" alt="Screenshot 2022-08-30 at 09 47 46" src="https://user-images.githubusercontent.com/383397/187392700-df043346-2cf3-4c5c-895a-ceb7006723ea.png">
 
 Above is what the vor looks like with javascript disabled.

Reply:
The class "js" was removed from the style